src/pages/support-commands/details/index.tsx

```diff
@@ -0,0 +1,212 @@
+  }, [userSubscription])
+
+  const onRefreshHistory = () => {
+    dispatch(fetchProfileRunHistory(window.location.href.split("/").pop()));
+  }
+
+  return (
+    <div>
+      <h2 className="px-9 my-0 flex justify-between">
+        {profile?.companyName}
+        {!impersonate ? (
+          <Button
+            title={`Impersonate as ${profile?.fullName}`}
+            className="cursor-pointer	text-sm inline-flex items-center bg-white normal-case"
+            onClick={() => {
+              dispatch(setImpersonate({ status: true, url: profile.id }));
+              // window.location.pathname = "/";
+            }}
+          >
+            <TheaterComedyIcon className="mr-2" /> Impersonate
+          </Button>
+        ) : null}
+      </h2>
+      <Grid container className="py-5 px-9" justifyContent="space-between">
+        <Card
+          key={'profile-card'}
+          index={0}
+          showEdit
+          onEdit={() => {
+            setOpen(true);
+            setTitle("profile");
+          }}
+          class="p-5 form-card"
+        >
+          <div className="img">
+            <img
+              src="https://avatars.githubusercontent.com/u/22358125?v=4"
+              alt="profile"
+              className="w-full"
+            />
+          </div>
+
+          <div className="details pl-7 inline-flex flex-col w-full">
+            <h3 className="m-0 mb-3">{profile?.companyName}</h3>
+            {!loadingDetails ? (
+              PROFILE_DETAILS_KEYS.map((key, index) => (
+                <span key={`profile-key-${index}`} className="text-gray-label mb-1">
+                  {profile?.[key] ? profile?.[key] : (<i className="text-[14px]">{formatToTitleCase(key)}</i>)}
+                </span>
+              ))
+            ) : (
+              <Box className="flex items-center justify-center p-8">
+                <CircularProgress color="secondary" />
+              </Box>
+            )}
+          </div>
+        </Card>
+
+        <Card key={'subscription-card'} index={1} class="p-5 form-card">
+          <div className="details inline-flex flex-col w-full">
+            <h3 className="m-0 mb-3">Current Subscription</h3>
+            {!loadingSubDetails ? (
+              userSubscription?.length && currentSubscription ? (
+                <>
+                  <span className={classNames("text-gray-label mb-1 tierType")}>
+                    {currentSubscription.type}
+                  </span>
+                  <span className={classNames("text-gray-label mb-1")}>
+                    Tier {currentSubscription.tierId}
+                  </span>
+                  <span className={classNames("text-gray-label mb-1")}>
+                    {dayjs(currentSubscription.startDate).format("MM/DD/YYYY")}
+                  </span>
+                  <span className={classNames("text-gray-label mb-1")}>
+                    {dayjs(currentSubscription.endDate).format("MM/DD/YYYY")}
+                  </span>
```

Comment:
I really dislike MM/DD/YYYY format but I guess it's a cultural thing ^^

Reply:
Fixed it to YYYY-MM-DD, and will never again put months first! :handshake:

Reply:
Thanks :D